Client/Frontend/Fakespot/FakespotOptInCardViewModel.swift

```diff
@@ -132,6 +134,22 @@ struct FakespotOptInCardViewModel {
         return finalString
     }
 
+    private func processStringBasedOnLocale() -> String {
+        var firstParagraph: String
+        switch Locale.current.languageCode {
```

Comment:
Instead of checking for locales here I think it would be better to add a var to `ShoppingProduct` that returns the supported websites with the same TLD as the current product. We would then use this var here to decide if we need the string with one (default) or 3 vendors.